nav2_bringup/launch/nav2_params.yaml

```diff
@@ -110,11 +111,15 @@ map_server:
     use_sim_time: True
     yaml_filename: "turtlebot3_world.yaml"
 
-nav2_controller:
+lifecycle_manager:
   ros__parameters:
     use_sim_time: True
 
-nav2_controller_lifecycle_client_node:
+lifecycle_manager_service_client:
```

Comment:
I'm not sure I follow where the additional node is here. There's a 'lifecycle_manager_service_client' and a 'lifecycle_manager_client_service_client' here in the parameters file, but when I look at the code, I only see this one:
https://github.com/ros-planning/navigation2/blob/f729650a8c0edf7df419fc4ad03730436235ae10/nav2_lifecycle_manager/src/lifecycle_manager_client.cpp#L28

Was this supposed to be renaming it?

Reply:
@mkhansen-intel 

No, it's correct, but I can see why it's confusing. It's confusing to me too :)

The **lifecycle_manager_service_client** is the node that the Lifecycle Manager uses when invoking services. I've used the convention with this type of node that is the parent node name + "_service_client". So, in this case, it's the "lifecycle_manager" + "_service_client". In this case it is calling the lifecycle services of the various nodes. 

The **lifecycle_manager_client_service_client** follows the same pattern - the lifecycle_manager_client has a node that it uses to invoke services (the services provided by the lifecycle_manager in this case). 

So, to make it clearer, I can change lifecycle_manager_client_service_client to simply be lifecycle_manager_client. How about that?